tests/test_intercept.js

```diff
@@ -1117,53 +861,22 @@ test('reply with file and pipe response', t => {
   )
 })
 
-test('reply with file with headers', t => {
-  let dataCalled = false
-
-  nock('http://www.filereplier2.com')
-    .get('/')
-    .replyWithFile(200, `${__dirname}/../assets/reply_file_2.txt.gz`, {
-      'content-encoding': 'gzip',
-    })
-
-  const req = http.request(
-    {
-      host: 'www.filereplier2.com',
-      path: '/',
-      port: 80,
-    },
-    res => {
-      t.equal(res.statusCode, 200)
-      res.on('end', () => {
-        t.ok(dataCalled)
-        t.end()
-      })
-      res.on('data', data => {
-        dataCalled = true
-        t.equal(data.length, 57)
```

Comment:
This assertion was changed because the old one does not make sense. 57 is the length of the file; 20 is its unzipped length. The old test would have passed without the content-encoding header. The point of sending the header is to signal to the client that the content should be unzipped on receipt.